.gitignore

```diff
@@ -1,11 +1,13 @@
 .DS_Store
 node_modules
 /build
-/.svelte-kit
+.svelte-kit
```

Comment:
I noticed `pnpm format` keeps touching stuff in here, not sure why -- not happening in another of my sk projects